src/Schema/Events/Tap.ts

```diff
@@ -292,6 +292,26 @@ export interface TappedConsign {
   subject: string
 }
 
+/**
+ * A user taps on an Artsy curated collection on a search tab
+ *
+ * This schema describes events sent to Segment from [[tappedCuratedCollection]]
+ *
+ * @example
+ * ```
+ * {
+ *   action: "tappedCuratedCollection",
+ *   context_module: "searchTab",
+ *   slug: "trending-this-week"
+ * }
+ * ```
+ */
+export interface TappedCuratedCollection {
+  action: ActionType.tappedCuratedCollection
+  context_module: ContextModule.searchTab
+  collection_slug: string
```

Comment:
Are there any other fields we want to track here? I can think of only a slug so that we know which collection performs better